assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-container.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React, { useState } from 'react'
+import {
+  NotesAndEventsTabView
+} from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-view'
+import {
+  useCreateNoteForElementMutation,
+  useDeleteNoteMutation,
+  useGetNotesForElementByTypeAndIdQuery
+} from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-api-slice.gen'
+import { useGlobalAssetContext } from '@Pimcore/modules/asset/hooks/use-global-asset-context'
+import { Result } from 'antd'
+import { Pagination } from '@Pimcore/components/pagination/pagination'
+import { useTranslation } from 'react-i18next'
+
+export const NotesAndEventsTabContainer = (): React.JSX.Element => {
+  const { t } = useTranslation()
+  const { context } = useGlobalAssetContext()
```

Comment:
Please use the normal asset context instead of the global one.
The global one is set a bit later during the render cycle and cost always some extra renders for the tabs.

I already adjusted it for the properties for example:
https://github.com/pimcore/studio-ui-bundle/blob/1.x/assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/properties/properties-container.tsx#L32